backend/routes/auth.js

```diff
@@ -262,6 +277,30 @@ module.exports = function (router) {
       }
     }
 
+    if (process.env.AWS_MARKETPLACE_DEPLOYMENT) {
+      /**
+       * In the case of a AWS marketplace deployment, the super-admin password
+       * is the EC2 instance ID.
```

Comment:
I'm not familiar with EC2. I assume there is no possibility an attacker could guess the instance ID?

Reply:
Not that I'm aware of.  Though I do think these should only be temporary passwords(GCP included).  Perhaps that's something we can fix forward.

Reply:
Yep agreed as a separate feature we should encourage (or maybe force) the admin to provide a new password the first time they login.

Reply:
See #473